velox/external/duckdb/duckdb-7.cpp

```diff
@@ -6959,9 +6959,9 @@ void ExpressionBinder::ReplaceMacroParametersRecursive(unique_ptr<ParsedExpressi
 		auto &sq = ((SubqueryExpression &)*expr).subquery;
 		ParsedExpressionIterator::EnumerateQueryNodeChildren(
 		    *sq->node, [&](unique_ptr<ParsedExpression> &child) { ReplaceMacroParametersRecursive(child); });
-		break;
```

Comment:
These is external vendor code , You might need to change code in ddb so as to not lose this when we import new versions. Read here on how to import ddb code : https://github.com/facebookincubator/velox/blob/5168694f10f778cbec1ea701c18b8c6e09a93419/velox/duckdb/README.md?plain=1#L4 .